app/screens/reactions/reactors_list/index.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+import {NativeScrollEvent, NativeSyntheticEvent, PanResponder} from 'react-native';
+import {FlatList} from 'react-native-gesture-handler';
+
+import {fetchUsersByIds} from '@actions/remote/user';
+import {useServerUrl} from '@context/server';
+
+import Reactor from './reactor';
+
+import type ReactionModel from '@typings/database/models/servers/reaction';
+
+type Props = {
+    reactions: ReactionModel[];
+}
+
+const ReactorsList = ({reactions}: Props) => {
+    const serverUrl = useServerUrl();
+    const [enabled, setEnabled] = useState(false);
+    const listRef = useRef<FlatList>(null);
+    const [direction, setDirection] = useState<'down' | 'up'>('down');
+    const prevOffset = useRef(0);
+    const panResponder = useRef(PanResponder.create({
+        onMoveShouldSetPanResponderCapture: (evt, g) => {
+            const dir = prevOffset.current < g.dy ? 'down' : 'up';
+            prevOffset.current = g.dy;
+            if (!enabled && dir === 'up') {
+                setEnabled(true);
+            }
+            setDirection(dir);
+            return false;
+        },
+    })).current;
+
+    const renderItem = useCallback(({item}) => (
+        <Reactor reaction={item}/>
+    ), [reactions]);
```

Comment:
hmm, why is `reactions` here and not `item`?

Reply:
cause the item is a reaction and item is only accessible in the callback????!?!?

Reply:
Then there is nothing to depend on. You can have the list of dependencies empty.

Reply:
That is true!